tube/mappings/etl.yaml

```diff
@@ -25,18 +25,13 @@ mappings:
           - name: ethnicity
           - name: vital_status
           - name: year_of_birth
-    filter:
-      name: filter_1
     aggregated_props:
       - name: _follow_ups_count
         path: follow_ups
         fn: count
-        filter: filter_1
       - name: _summary_drug_uses_count
         path: follow_ups.summary_drug_uses
         fn: count
-        filter: aliquots
```

Comment:
I was only talking about removing the `named filters` thanh, don't we have to place at least one filter example here? Moreover, I was having a little difficulty identifying where in code do we use this [elt.yaml](https://github.com/uc-cdis/tube/blob/master/tube/mappings/etl.yaml)? I was trying to identify but couldn't find it. Would you be able to guide me here?

Reply:
we no longer use this file @nss10 . Just put here as an example